### ISSUE TITLE
PS2 Redump DAT Processing Issue

### ISSUE THREAD

**Issue by Hooterman** [unexpectedpanda/retool#291]
During processing of dat file, getting the following error:
```
Traceback (most recent call last):
  File "retoolgui.py", line 225, in run
  File "retool.py", line 224, in main
  File "modules\titletools.py", line 614, in includes
KeyError: 'psyvariar - complete edition'
``` 
Dat file:
[Sony - PlayStation 2 - Datfile (11345) (2023-11-29 15-48-43).zip](https://github.com/unexpectedpanda/retool/files/13537652/Sony.-.PlayStation.2.-.Datfile.11345.2023-11-29.15-48-43.zip)

**Operating system**
Windows 11

**Retool edition**
retool-gui.exe

**Retool version**
v2.01.6

**Comment by Hooterman** [comment]
In my Overrides tab I have the following:
```
Beta)
(Beta 1)
(Beta 2)
(Beta 3)
Alpha)
(Proto
```

**Comment by unexpectedpanda** [comment]
Okay, this will be fixed in 2.01.7.
